### ISSUE TITLE
linked images of controllers are missing

### ISSUE THREAD

**Issue by Be-ing** [mixxxdj/manual#133]
Images linked from the controller documentation on the wiki are missing from the manual.

**Comment by Holzhaus** [comment]
If you're talking about the controller images at the top: I removed them. Adding a photo of every controller to the manual doesn't add any value (you already know how your controller looks like), but makes the manual in file size. I also doubt we have the necessary copyright to use professional product photos from the vendor.

**Comment by Be-ing** [comment]
> Adding a photo of every controller to the manual doesn't add any value (you already know how your controller looks like)

No, it is essential. Many of the mapping descriptions refer to a diagram.

**Comment by Holzhaus** [comment]
> > Adding a photo of every controller to the manual doesn't add any value (you already know how your controller looks like)
> 
> No, it is essential. Many of the mapping descriptions refer to a diagram.

Yes, that's why I kept the diagrams. I agree that adding such a diagram (ideally vector images) should be best practice for manual pages, e. g.:
- https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_200.html 
- https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html

But an unlabeled photo of a controller is not helpful because you have it in front of you. Also, we need to have copyright issues in mind if we don't want to have our repo DCMA'd and taken down. 

> Furthermore, reading a bunch of text describing how a mapping works is not very helpful for someone trying to decide what controller to buy if they can't see what they're reading about.

That's a different issue. A shopping guide should go into the wiki instead of the manual IMHO, maybe categorized into 2-deck, 4-deck and single-deck controllers. Let's not conflate this with "I have this controller and want to know how the Mixxx mapping works".

Also, if you plan to purchase a specific controller it's easy to google a photo of it.
